packages/meta-harness/lib/meta_harness_common.py

```
@@ -0,0 +1,943 @@
+# store パス解決
+# ---------------------------------------------------------------------------
+
+STORE_SUBDIRS = ("candidates", "runs", "locks", "tmp", "rejected", "reports")
+
+# overlay/config-patch.json（存在する場合のみ）は overlay の facets/** コンテンツではなく
+# Sec1-8 の予約サイドカーファイル。overlay_files 一覧や facets/ prefix 検証の対象外にする。
+CONFIG_PATCH_FILENAME = "config-patch.json"
+
+
+def store_dir(main_root: Path, config: dict) -> Path:
+    """store ルート（既定 `.claude/meta-harness`）の絶対パスを返す。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)